tests/queries/0_stateless/01813_use_local_shard_if_no_remote_connections.sql

```diff
@@ -0,0 +1,19 @@
+DROP DATABASE IF EXISTS 01813_db;
+CREATE DATABASE 01813_db;
+
+DROP TABLE IF EXISTS 01813_db.data;
+CREATE TABLE 01813_db.data (a Int64, b Int64) ENGINE = TinyLog();
+
+DROP TABLE IF EXISTS 01813_db.data_distributed;
+CREATE TABLE 01813_db.data_distributed (a Int64, b Int64) ENGINE = Distributed(test_shard_localhost, '01813_db', data);
```

Comment:
If you specify empty db here, it should use the default one, so you won't have to specify an explicit database.

Reply:
Thanks.